1.Normalize/normalization-pipeline.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Saketh Mynampati 2020
+# Batch Effect Correction Benchmarking
+# Pipeline to Process Data
+#
+# 1) Normalize Single Cell Data
+# 2) Select Features
+
+set -e
+
+# Step 0: Convert all notebooks to scripts and create CSVs of the data using R
+jupyter nbconvert --to=script \
+        --FilesWriter.build_directory=../scripts/nbconverted \
+        *.ipynb
+
+cd ..
+mkdir -p data
+cd "1.Normalize/"
+
+Rscript "0.Create-csvs.R" $1 $2 
```

Comment:
I am not seeing how this is executing - maybe I am missing something? What is $1 and $2?